src/data_operations.py

```
@@ -0,0 +1,100 @@
+import pandas as pd
+import time
+from datetime import datetime
+from io import StringIO
+import pickle
+from fredapi import Fred
+
+def process_non_sofr_data(data_map_dict, fred, col_date, dataPath, SAVE_AS_PICKLE, PUSH_TO_GCP, bucket):
```

Ruff (0.8.2)

Missing return type annotation for public function `process_non_sofr_data`

(ANN201)